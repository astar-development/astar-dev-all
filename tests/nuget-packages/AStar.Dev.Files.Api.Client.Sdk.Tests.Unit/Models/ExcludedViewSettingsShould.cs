using AStar.Dev.Files.Api.Client.SDK.Models;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

public sealed class ExcludedViewSettingsShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new ExcludedViewSettings().ToString().ShouldMatchApproved();
}
