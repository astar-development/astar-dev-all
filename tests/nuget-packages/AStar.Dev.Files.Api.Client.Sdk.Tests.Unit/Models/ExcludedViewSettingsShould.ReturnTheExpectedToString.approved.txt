{
  "excludeViewedPeriodInDays": 7,
  "excludeViewed": true
}